### ISSUE TITLE
Unify and organize how filters are documented in the `TracingPolicy` documentation

### ISSUE THREAD

**Issue by mtardy** [cilium/tetragon#708]
The way filters (`matchArgs`, `matchPIDs`, etc...) are documented in a heterogeneous manner, we could think of a standard way of documenting each filter, one suggestion could be:

1. introduction
2. use case
3. inputs, values, options, and operators supported
4. limitations
5. examples

https://tetragon.io/docs/concepts/tracing-policy/selectors/

**Comment by JoonsooWon** [comment]
@mtardy 
Hello. I'm interested in working on this issue.
Is this issue still active and open for contributioin?

**Comment by kazuru-chidumbwe** [comment]
Hello @mtardy 
I would like to take this. 

Plan: introduce a consistent filter doc template (intro · use case · inputs/operators · limitations · examples) and apply it first to one selector (matchArgs or matchPIDs), then follow up for the others. 

Please assign if that works.

**Comment by mtardy** [comment]
Yes sorry that I never replied btw @JoonsooWon, just seeing this.

This might be some amount of work to get it right, just sayin but yes @kazuru-chidumbwe feel free.
